I have included "sceAudioRouting.h" in HLETables.cpp ,not sure broke linux build

996lib/libCore.a(HLETables.cpp.o): In function `RegisterAllModules()':

997HLETables.cpp:(.text+0x191): undefined reference to `Register_sceAudioRouting()'

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1172,6 +1172,8 @@ add_library(${CoreLibName} ${CoreLinkType}
 	Core/HLE/sceAudio.cpp
 	Core/HLE/sceAudiocodec.cpp
 	Core/HLE/sceAudiocodec.h
+	Core/HLE/sceAudioRouting.cpp
+	Core/HLE/sceAudioRouting.h
 	Core/HLE/sceAudio.h
 	Core/HLE/sceCcc.h
 	Core/HLE/sceCcc.cpp

--- a/Core/CMakeLists.txt
+++ b/Core/CMakeLists.txt
@@ -32,6 +32,7 @@ set(SRCS
   HLE/sceAtrac.cpp
   HLE/__sceAudio.cpp
   HLE/sceAudio.cpp
+  HLE/sceAudioRouting.cpp
   HLE/sceCcc.cpp
   HLE/sceChnnlsv.cpp
   HLE/sceCtrl.cpp

--- a/Core/HLE/HLETables.cpp
+++ b/Core/HLE/HLETables.cpp
@@ -338,5 +338,6 @@ void RegisterAllModules() {
 	Register_sceAdler();
 	Register_sceSfmt19937();
 	Register_sceAudioRouting();
+	Register_sceUsbCam();
 }
 

